src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Queries/ChildCategoriesQueryResponseType.cs

```diff
@@ -8,7 +8,7 @@ public class ChildCategoriesQueryResponseType : ExtendableGraphType<ChildCategor
 {
     public ChildCategoriesQueryResponseType()
     {
-        Field<ListGraphType<CategoryType>>(
+        Field<NonNullGraphType<ListGraphType<NonNullGraphType<CategoryType>>>>(
```

Comment:
add empty ChildCategories initialization by default in ChildCategoriesQueryResponse.cs

Reply:
Done